### PR TITLE
feat: added lists and navigation component

### PR DESCRIPTION
**Pull request by rahulk31**
Created Lists and Navigation Component.

### CODE REVIEW

**Review by thenorthkun — APPROVED** [review]
- Looks like navigation.css is rather empty or not visible
- UI suggestion : You can adjust the padding of the **Documentation** button similar to **Get Started** button ie., `padding: 0.8rem 1.5rem` so that they both look visually aligned.

Loved the overall UI though, good selection of font 👍

**Comment by rahulk31** [comment]
> * Looks like navigation.css is rather empty or not visible
> * UI suggestion : You can adjust the padding of the **Documentation** button similar to **Get Started** button ie., `padding: 0.8rem 1.5rem` so that they both look visually aligned.
> 
> Loved the overall UI though, good selection of font 👍

Hey Dhruv, thanks for the review and the suggested fix. I have committed the changes. :)

File: css/documentation.css [hunk 1]
```diff
@@ -139,6 +139,11 @@ a {
   flex-wrap: wrap;
 }
 
+#navigation .component-examples {
```
Comment (Naman-Saxena1) [inline css/documentation.css]: If you can use only class that would be better, using id increases its specificity, would be tough to override it in the future if you are using Component Library in other projects.

**Review by Naman-Saxena1 — APPROVED** [review]
Overall code quality and syntax looks good 👍

File: css/documentation.css (hunk 1 above)
Comment (rahulk31) [reply css/documentation.css]: Hey Naman, thanks for pointing this out. I have fixed it and committed the change. :)

**Review by pradipgharge — COMMENTED** [review]
Looks good to merge !

**Pull request merged**